src/components/SchemaViewer/index.jsx

```diff
@@ -35,7 +44,11 @@ function SchemaViewer({ schema }) {
       {sourceMode ? (
         <SourceView schema={schema} />
       ) : (
-        <SchemaTable schemaTree={schemaTree} setSchemaTree={setSchemaTree} />
+        <SchemaTable
+          schemaTree={schemaTree}
+          setSchemaTree={setSchemaTree}
+          references={referenceMap}

```

Comment:
Do we have an example in the storybook where the schema is not able to find the reference? I think that would be good to have.